bin/strings.json

```diff
@@ -235,10 +235,10 @@
         "GEN_LIBS_ARG_CLEAN" : "Remove the 'prebuilt' folder at first.",
         "GEN_LIBS_ARG_ENGINE" : "Specify the engine path. Default is the engine root path of current tools.",
         "GEN_LIBS_ARG_PLATFORM" : "Specify the target platform. Can specify multi platform by using '-p' multi times. Default generate all available platforms.",
-        "GEN_LIBS_ARG_MODE" : "Generate cocos libs for debug or release. Default is release.",
+        "GEN_LIBS_ARG_MODE" : "Generate cocos libs for debug or release. Default is debug.",
```

Comment:
review the code in `gen_libs.py`,
```
parser.add_argument('-m', "--mode", dest='compile_mode', default='debug', choices=['debug', 'release'],
                            help=MultiLanguage.get_string('GEN_LIBS_ARG_MODE'))
```
default value is debug, not release, so change the help info